index.html

```diff
@@ -1,192 +1,192 @@
-                <li><a href="https://www.linkedin.com/danyaugustin/"><img src="img/linkedin.svg" alt="linkedin-icon"></a></li>
-                <li><a href="https://github.com/amiralda"><img src="img/github.svg" alt="github-icon"></a></li>
-                <li><a href="https://www.medium.com/amiralda/"><img src="img/medium.svg" alt="medium-icon"></a></li>
-                <li><a href="https://angel.co/u/dany-augustin"><img src="img/angellist.svg" alt="angellist-icon"></a></li>
-            </ul>
-    </section>
+                <h2 class="connect">LET'S CONNECT</h2>
+                <ul class="icon-bio">
+                    <li><a href="https://twitter.com/amiraldany"><img src="img/twitter.svg" alt="twitter-icon"></a></li>
+                    <li><a href="https://www.linkedin.com/in/dany-augustin/"><img src="img/linkedin.svg" alt="linkedin-icon"></a></li>
+                    <li><a href="https://github.com/amiralda"><img src="img/github.svg" alt="github-icon"></a></li>
+                    <li><a href="https://medium.com/@danyaugustin1982"><img src="img/medium.svg" alt="medium-icon"></a></li>
+                    <li><a href="https://angel.co/u/dany-augustin"><img src="img/angellist.svg" alt="angellist-icon"></a></li>
+                </ul>
+        </section>
 
-    <section class="Parent">
-        <article class="box">
-            <img class="image" src="img/tonic.png" alt="tonic">
-            <div class="box-info">
-                <h2 class="title">Tonic </h2>
-                <div class="description">
-                    <a href="#" class="client">CANOPY</a>
-                    <img class="dot" src="img/dot.png" alt="dot">
-                    <a href="#" class="tech">BACK END DEV</a>
-                    <img class="dot" src="img/dot.png" alt="dot">
-                    <a href="#" class="year">2015</a>
+        <section class="Parent">
+            <article class="box">
+                <img class="image" src="img/tonic.png" alt="tonic">
+                <div class="box-info">
+                    <h2 class="title">Tonic </h2>
+                    <div class="description">
+                        <a href="#" class="client">CANOPY</a>
+                        <img class="dot" src="img/dot.png" alt="dot">
+                        <a href="#" class="tech">BACK END DEV</a>
+                        <img class="dot" src="img/dot.png" alt="dot">
+                        <a href="#" class="year">2015</a>
+                    </div>
+                    <p class="details">A daily selection Of privately personalized reads; no accounts or sign-ups required
+                    </p>
+                    <ul class="stack">
+                        <li class="style">html</li>
+                        <li class="style">css</li>
+                        <li class="style">javascript</li>
+                    </ul>
+                    <a href="#" class="preview">See Project</a>
                 </div>
```

Comment:
- [ ] Kindly ensure that you use the spacing elements and units as recommended by the Figma design on all your project cards. Elements like `padding`, `margin`, `gap`, `height`, `width`, etc.
  - [ ] Also, your usage of fonts, kindly stick to the figma's recommendation. You might have to download some of those web fonts from [Google Fonts](fonts.google.com) and import them into your file :thinking:

When you have enough space in your design, your design becomes very easy to interact with

|Your Design| Figma Design|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/222831629-b1acbce2-0e1d-4034-a477-85d3b6341f83.png)|![image](https://user-images.githubusercontent.com/99715640/222831710-6d5ee9c0-026b-4f15-b12e-fb681e55a8e3.png)|

Reply:
Here you can review the width of the children elements and use a flexbox row with <sub> tag linked with a CSS class that groups them for use inside a media display.

on the outside you put display: none;

I like to make many media tags in the CSS to make development faster and easier to locate for adjustments and changes.